dont compile libpci staticly,
but use the system libpci shared library.

diff -Nur pciutils-2.2.1.orig/Makefile pciutils-2.2.1/Makefile
--- pciutils-2.2.1.orig/Makefile	Sat Nov 26 15:00:57 2005
+++ pciutils-2.2.1/Makefile	Fri Jun 23 09:22:23 2006
@@ -14,7 +14,8 @@
 MANDIR:=$(shell if [ -d $(PREFIX)/share/man ] ; then echo $(PREFIX)/share/man ; else echo $(PREFIX)/man ; fi)
 INSTALL=install
 DIRINSTALL=install -d
-PCILIB=lib/libpci.a
+PCILIB=
+LDFLAGS+=-lpci
 PCIINC=lib/config.h lib/header.h lib/pci.h lib/types.h lib/sysdep.h
 
 ifeq ($(shell uname),NetBSD)
